internal/provider/provider.go

```diff
@@ -126,14 +126,15 @@ func (p *fluxProvider) Schema(ctx context.Context, req provider.SchemaRequest, r
 			},
 			"config_context": schema.StringAttribute{
 				Optional: true,
+				Description: "Context to choose from the config file. Can be sourced from `KUBE_CTX`.",
 			},
 			"config_context_auth_info": schema.StringAttribute{
 				Optional:    true,
-				Description: "",
+				Description: "Authentication info context of the kube config (name of the kubeconfig user, `--user` flag in `kubectl`). Can be sourced from `KUBE_CTX_AUTH_INFO`.",
```

Comment:
These cannot be source from environment variables currently.

Reply:
Updated to align with functionality.

Reply:
Updated to align with current functionality.